src/app/models/propTypes/inline/index.js

```diff
@@ -5,7 +5,6 @@ import { inlineLinkBlockPropTypes } from '../inlineLink';
 
 export const inlineModelPropTypes = {
   language: string.isRequired,
-  text: string.isRequired,
```

Comment:
Text is required for the `aria-label` for the InlineLink. See here: https://github.com/bbc/simorgh/blob/latest/src/app/containers/InlineLink/index.jsx#L35

Reply:
In that case, why is text not being used in the container if it is still being passed down to it?

Reply:
@sareh src/app/containers/InlineContainer/index.jsx doesnt use `text` though, but it uses `inlineModelPropTypes` as a prop type, so it 'requires' a prop it never uses. Any idea whats wrong? Were not sure if the code is wrong, or the prop types

Reply:
The `inlineModelPropTypes` are not complete. 
`inlineContainer/index.jsx` requires `blocks` - specifically `blocks[0].model.text` for the inlikeLink text for the aria-label.

Reply:
This isnt `blocks[0].model.text` though, its just `text` at the top level (as far as I can see) :P. Is it needed at the top level outside of blocks?

Reply:
Talked to sareh about this, confirmed that we're getting rid of `text` at the top level and not `blocks[0].model.text` so this is fine.